physlr/physlr.py

```diff
@@ -1345,6 +1418,8 @@ def determine_molecules(g, u, strategy):
             communities = Physlr.determine_molecules_bc_louvain(g, u)
         elif strategy == 4:  # bi-connected + sqCos
             communities = Physlr.determine_molecules_bc_cosine_of_squared(g, u)
+        elif strategy == 10:  # bi-connected + partition + bi-connected + k-cliques + merge
```

Comment:
```suggestion
        elif strategy == 5:  # bi-connected + partition + bi-connected + k-cliques + merge
```
5?

Reply:
um, I thought we might add more single-approach strategies later. So I reserved 5-9 for them. At the end of the day (the end of the year better applies here :)) ) we can change it to the a better number.

Reply:
Perhaps in a separate PR, I'd prefer to name the strategies rather than number them. Like `bic` and `k3` and `louvain`.

Reply:
hum good idea.